.github/checkgroup.yml

```diff
@@ -108,16 +118,16 @@ subprojects:
 
   - id: "lightning_app"
     paths:
-      - ".azure/app-cloud-e2e.yml"
       - "requirements/app/**"
       - "src/lightning_app/**"
       - "tests/tests_app/**"
       - "tests/tests_app_examples/**"
-      - "tests/tests_clusters/**"
       # the examples are used in the app CI
       - "examples/app_*"
     checks:
-      - "App.cloud-e2e"
```

Comment:
This comment: https://github.com/Lightning-AI/lightning/pull/14282#discussion_r950669432 was incorrect